merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/PermissionsRepository.java

```diff
@@ -0,0 +1,84 @@
+package gov.nasa.jpl.aerie.merlin.server.remotes.postgres;
+
+import gov.nasa.jpl.aerie.merlin.server.models.DatasetId;
+import gov.nasa.jpl.aerie.merlin.server.models.PlanId;
+import gov.nasa.jpl.aerie.merlin.server.remotes.PermissionType;
+
+import javax.sql.DataSource;
+import java.sql.SQLException;
+
+public final class PermissionsRepository {
```

Comment:
Is this `PermissionsRepository` setting a new precedent that we should "flatten" interfaces like `ResultsCellRepository` to only the postgres implementation since `InMemory*` and `Stub*` aren't being effectively utilized?